apps/fluent-tester/src/TestComponents/Switch/E2ESwitchTest.tsx

```diff
@@ -1,19 +1,25 @@
 import * as React from 'react';
 import { Stack } from '@fluentui-react-native/stack';
 import { Switch } from '@fluentui-react-native/switch';
-import { Text } from 'react-native';
```

Comment:
Related: I think React Native macOS has the iOS fix to make text white in dark mode, and I need to go upstream that 😅

Reply:
That would be great to have 😀